src/components/Person/Person.jsx

```diff
@@ -1 +1,44 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
```

Comment:
```suggestion
      <h2 className="Person__name">{`My name is ${name}`}</h2>
      
      {age && (
```

Add indents between nested siblings.